core/flow/tc_warm_up.go

```diff
@@ -41,64 +41,67 @@ func NewWarmUpTrafficShapingCalculator(rule *FlowRule) *WarmUpTrafficShapingCalc
 		maxToken:          maxToken,
 		slope:             slope,
 		threshold:         rule.Count,
-		storedTokens:      new(uint64),
-		lastFilledTime:    new(uint64),
+		storedTokens:      0,
```

Comment:
The default value is 0 and no assignment is required。

Reply:
Yes, just an explicit declaration